exchange_calendars/exchange_calendar_xtai.py

```diff
@@ -222,17 +270,17 @@ def bridge_fri(dt: datetime.datetime) -> datetime.datetime | None:
 
 tomb_sweeping_day = manual_nearest_workday(qingming_festival_dates)
 
-tomb_sweeping_day_extras = manual_extra_days(tomb_sweeping_day)
+tomb_sweeping_day_extras = evalute_tomb_sweeping_extra_days(qingming_festival_dates)
 
 dragon_boat_festival = manual_nearest_workday(dragon_boat_festival_dates)
 
-dragon_boat_festival_extras = manual_extra_days(dragon_boat_festival)
+dragon_boat_festival_extras = manual_extra_days_07_thr_2023(dragon_boat_festival)
 
 mid_autumn_festival = manual_nearest_workday(
     mid_autumn_festival_dates,
 )
 
-mid_autumn_festival_extras = manual_extra_days(mid_autumn_festival)
+mid_autumn_festival_extras = manual_extra_days_07_thr_2023(mid_autumn_festival)
```

Comment:
(as for the 'dragon_boat_festival'...) **Can you confirm that from 2024 the 'mid_autumn_festival' will not pass from, for example, a Sunday to the following Monday.**